src/Background/NotificationRequest.vala

```diff
@@ -3,16 +3,44 @@
  * SPDX-License-Identifier: LGPL-2.1-or-later
  */
 
+[DBus (name = "org.freedesktop.Notifications")]
+private interface Fdo.Notifications : Object {
+    public signal void action_invoked (uint32 id, string action_key);
+    public signal void notification_closed (uint32 id, CloseReason reason);
+
+    public const string NAME = "org.freedesktop.Notifications";
+    public const string PATH = "/org/freedesktop/Notifications";
+
+    [DBus (signature = "u")]
```

Comment:
I get this warning: `Dropping signal NotificationClosed of type (uu) since the type from the expected interface is (ui)`
No idea why though 🤷